core/src/main/java/io/grpc/inprocess/InProcessTransport.java

```diff
@@ -78,6 +80,9 @@
 
   public InProcessTransport(String name) {
     this.name = name;
+    this.serverStreamAttributes = Attributes.newBuilder()
+        .set(ServerCall.REMOTE_ADDR_KEY, new GrpcUtil.InProcessSocketAddress(name))
```

Comment:
Ah yes, that makes perfect sense. Updated.

On Fri, Mar 25, 2016 at 4:34 PM, Joshua Humphries <notifications@github.com>
wrote:

> In core/src/main/java/io/grpc/inprocess/InProcessTransport.java
> <https://github.com/grpc/grpc-java/pull/1593#discussion_r57455118>:
>
> > @@ -78,6 +80,9 @@
> >
> >    public InProcessTransport(String name) {
> >      this.name = name;
> > +    this.serverStreamAttributes = Attributes.newBuilder()
> > +        .set(ServerCall.REMOTE_ADDR_KEY, new GrpcUtil.InProcessSocketAddress(name))
>
> I think it should be a top-level type in the inprocess package.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc-java/pull/1593/files/35b89ebd8f30a03fc0ef98809790d226a9ee7790#r57455118>
>